documentation/src/main/asciidoc/user_guide/server_tasks.adoc

```diff
@@ -0,0 +1,5 @@
+=== Server Tasks
+
+TODO: This section needs to be completed still
```

Comment:
I've created [ISPN-7935](https://issues.jboss.org/browse/ISPN-7935) to address that. I'll fix that once its dependencies are fixed.